### PR TITLE
Add ansible playbook for setting up e2e test

### PR DESCRIPTION
**Pull request by ifigotin**
Add ansible playbook that will set up an end to end test. The paybook will do the following:
- install java, maven 3, and other packages onto a predefined Google Cloud Compute instance (see host, create_vm.sh)
- get the latest gcloud-maven-plugin
- get and install the latest gcloud SDK
- deploy a java test application (mvn gcloud:deploy) to App Engine

Note: the creation of Google Compute Engine VM can also be automated in the future. Also, the verification of successful deployment will be implemented later.

### CODE REVIEW

File: e2e/hosts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[local]
+localhost
+
+[compute]
+104.197.110.30 ansible_ssh_private_key_file=/Users/ifigotin/.ssh/google_compute_engine
```
Comment (ludoch) [inline e2e/hosts]: remove user specific data

**Comment by ifigotin** [comment]
done

File: e2e/hosts (hunk 1 above)
Comment (ifigotin) [inline e2e/hosts]: Done.

**Comment by ludoch** [comment]
LGTM

**Pull request merged**